### ISSUE TITLE
Add mention of `decode_responses` to README.md

### ISSUE THREAD

**Issue by jac08h** [underyx/flask-redis#50]
I was wondering if the FlaskRedis can take `decode_responses` keyword argument same way as **redis-py** package and apparently it can. I believe itwould be handy if this information could be found in the README.md.

On the other hand, perhaps it is possible to deduce it from 
> The redis client you created above from FlaskRedis acts just like a regular Redis instance from the redis-py library:

What do you think?

**Comment by underyx** [comment]
Hey, thanks for mentioning this! I think mentioning `decode_responses` by name is a bit overkill maybe, and there's no easy way to tell which parameters should be explicitly written in the README.

But you raise a good point that the current wording is not clear enough! Let me rephrase it real quick and link to the relevant docs.
